### PR TITLE
feat(cli): Added example to admin-cluster.go and projectwindow.go files

### PR DESCRIPTION
**Pull request by surajyadav1108**
part of the issue #15690 
and part of the PR #16103 

resolved issues 

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] The title of the PR conforms to the [Toolchain Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/toolchain-guide/#title-of-the-pr)
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).
* [x] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.

### CODE REVIEW

**Review by crenshaw-dev — DISMISSED** [review]
I think many of these examples rely too heavily on demonstrating global flags. Can you rework the examples to rely only on flags specific to the command?

**Comment by surajyadav1108** [comment]
@crenshaw-dev hmm looking back to it yes you are correct,  I will do the required changes

**Comment by surajyadav1108** [comment]
@crenshaw-dev made the changes.

File: cmd/argocd/commands/admin/cluster.go [hunk 1]
```diff
@@ -492,6 +510,19 @@ func NewClusterConfig() *cobra.Command {
...
+argocd admin cluster kubeconfig my-cluster
+
+#listing available kubeconfigs for clusters managed by argocd
+argocd admin cluster kubeconfig
+
+#removing a specific kubeconfig file 
+argocd admin cluster kubeconfig my-cluster --delete
+
+#Generate a Kubeconfig for a Cluster with TLS Verification Disabled
+argocd admin cluster kubeconfig https://cluster-api-url:6443 /path/to/output/kubeconfig.yaml --insecure-skip-tls-verify`,
```
Comment (ishitasequeira) [inline cmd/argocd/commands/admin/cluster.go:525]: Please follow the same format for the comments of the command. For example, keep first character capital everywhere.

File: cmd/argocd/commands/projectwindows.go [hunk 2]
```diff
@@ -22,6 +22,18 @@ func NewProjectWindowsCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
...
+argocd proj windows add my-project \
+--schedule "0 0 * * 1-5" \
+--duration 3600 \
+--prune
+
+#- Delete a sync window from a project. 
+argocd proj windows delete <project-name> <window-id>
+
+#list project sync windows
+argocd proj windows list <project-name>`,
```
Comment (ishitasequeira) [inline cmd/argocd/commands/projectwindows.go:36]: Same comment here. Keep uniform formatting.

File: cmd/argocd/commands/projectwindows.go [hunk 3]
```diff
@@ -79,6 +97,15 @@ func NewProjectWindowsEnableManualSyncCommand(clientOpts *argocdclient.ClientOpt
...
 		Long:  "Enable manual sync for a sync window. Requires ID which can be found by running \"argocd proj windows list PROJECT\"",
+		Example: `
+#enabling manual sync for a general case
+argocd proj windows enable-manual-sync PROJECT ID 
+
+#enabling manual sync for a windows set on the default project with Id 2
+agrocd proj windows enable-manual-sync default 2
+
+# Enabling manual synchronization with a custom message
+argocd proj windows enable-manual-sync my-app-project --message "Manual sync initiated by admin."`,
```
Comment (ishitasequeira) [inline cmd/argocd/commands/projectwindows.go:108]: Same here.

**Review by ishitasequeira — CHANGES_REQUESTED** [review]

File: cmd/argocd/commands/projectwindows.go [hunk 4]
```diff
@@ -42,6 +54,12 @@ func NewProjectWindowsDisableManualSyncCommand(clientOpts *argocdclient.ClientOp
 		Use:   "disable-manual-sync PROJECT ID",
 		Short: "Disable manual sync for a sync window",
 		Long:  "Disable manual sync for a sync window. Requires ID which can be found by running \"argocd proj windows list PROJECT\"",
+		Example: `
+#Disable manual sync for a sync window for the Project 
+argocd proj windows disable-manual-sync PROJECT ID 
+
+#Disbaling manual sync for a windows set on the default project with Id 0
+agrocd proj windows disable-manual-sync default 0`,
```
Comment (chengfang) [inline cmd/argocd/commands/projectwindows.go:62]: command name should be `argocd`

File: cmd/argocd/commands/projectwindows.go [hunk 5]
```diff
@@ -79,6 +97,15 @@ func NewProjectWindowsEnableManualSyncCommand(clientOpts *argocdclient.ClientOpt
 		Use:   "enable-manual-sync PROJECT ID",
 		Short: "Enable manual sync for a sync window",
 		Long:  "Enable manual sync for a sync window. Requires ID which can be found by running \"argocd proj windows list PROJECT\"",
+		Example: `
+#Enabling manual sync for a general case
+argocd proj windows enable-manual-sync PROJECT ID 
+
+#Enabling manual sync for a windows set on the default project with Id 2
+agrocd proj windows enable-manual-sync default 2
```
Comment (chengfang) [inline cmd/argocd/commands/projectwindows.go:105]: command name should be `argocd`

File: cmd/argocd/commands/projectwindows.go (hunk 4 above)
Comment (surajyadav1108) [reply cmd/argocd/commands/projectwindows.go]: man

**Review by chengfang — APPROVED** [review]

File: cmd/argocd/commands/admin/cluster.go [hunk 6]
```diff
@@ -492,6 +510,19 @@ func NewClusterConfig() *cobra.Command {
 		Use:               "kubeconfig CLUSTER_URL OUTPUT_PATH",
 		Short:             "Generates kubeconfig for the specified cluster",
 		DisableAutoGenTag: true,
+		Example: `
+
+#Generate a kubeconfig for a cluster named "my-cluster" on console
+argocd admin cluster kubeconfig my-cluster
+
+#listing available kubeconfigs for clusters managed by argocd
```
Comment (ishitasequeira) [inline cmd/argocd/commands/admin/cluster.go:518]:
```suggestion
#Listing available kubeconfigs for clusters managed by argocd
```

File: cmd/argocd/commands/projectwindows.go [hunk 7]
```diff
@@ -22,6 +22,18 @@ func NewProjectWindowsCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
...
+#Add a sync window to a project
+argocd proj windows add my-project \
+--schedule "0 0 * * 1-5" \
+--duration 3600 \
+--prune
+
+#Delete a sync window from a project 
+argocd proj windows delete <project-name> <window-id>
+
+#list project sync windows
```
Comment (ishitasequeira) [inline cmd/argocd/commands/projectwindows.go:35]:
```suggestion
#List project sync windows
```

**Review by ishitasequeira — CHANGES_REQUESTED** [review]

File: cmd/argocd/commands/admin/cluster.go (hunk 6 above)
Comment (surajyadav1108) [reply cmd/argocd/commands/admin/cluster.go]: never been this  blind

**Comment by surajyadav1108** [comment]
@ishitasequeira  build test UI code is  failing even  after retriggering. You can try a manual build to check for errors. I haven't changed anything apart from correcting a spelling mistake, so this check should pass.

**Comment by surajyadav1108** [comment]
@ishitasequeira ok found out why the Build test  UI code was failing and fixed it but after updating the branch by pulling in from upstream there were some issues in code that has already been merged into upstream master and that's why my master update check fails for lint and tests but there is a open PR correcting this issue  
#16275  I believe after that PR merge that will resolve these errors  too.

**Review by ishitasequeira — APPROVED** [review]
@surajyadav1108 , Please rebase the PR again. The PR #16275 is merged.

**Comment by surajyadav1108** [comment]
@ishitasequeira 
It seems like make lint-ui passes with no issues on my local clone, but it keeps failing here, and I'm not sure why it's happening repeatedly. Instead of making numerous failing commits, I suggest closing this for now. I'll delete my forked repo and reopen a PR after some time. or I can open a new PR now if you say so  what do you suggest?

**Comment by surajyadav1108** [comment]
Updating the master branch had no issues is it should be. @ishitasequeira

**Review by ishitasequeira — APPROVED** [review]
Thanks @surajyadav1108 !! LGTM!!

**Comment by surajyadav1108** [comment]
@ishitasequeira is there anything stopping auto-merge?

**Review by ishitasequeira — APPROVED** [review]
Thanks @surajyadav1108 for addressing all the comments. Will merge the PR.

**Pull request merged**